.drone.yml

```diff
@@ -765,6 +768,25 @@ steps:
   - name: cache
     path: /tmp/artifacts
 
+- name: upload to codecov
+  image: robertstettner/drone-codecov
```

Comment:
I love images that just are just from some dude

Reply:
the repo hasn't been touched in 6 years, is the image also that old!?

Reply:
It looks like it's also just a simple wrapper around the deprecated bash uploader; I wonder if we should just toss together a proper uploader that uses the new one from https://docs.codecov.com/docs/codecov-uploader

Reply:
I'll open a future PR to move to directly uploading via their bash tool

Reply:
no, bash is bad, we want to use the new non-bash one.